java/README.md

```diff
@@ -0,0 +1,56 @@
+# Spark Rapids ML Java
+
+Spark-rapids-ml java project is the java wrapper of spark-rapids-ml python package,
+which could accelerate the Spark ML library on the connect environment without changing
+users' code.
+
+## Compile
+
+``` shell
+mvn clean package
+```
+
+After compiling, you can get the latest `com.nvidia.rapids.ml-1.0-SNAPSHOT.jar` under target
```

Comment:
Would be good to explain more clearly how to compile until spark 4.0 snapshot dependencies are in maven central.

Reply:
Done.